sys-process/btop/btop-1.4.0.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake optfeature toolchain-funcs xdg
+
+DESCRIPTION="A monitor of resources"
+HOMEPAGE="https://github.com/aristocratos/btop"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/aristocratos/btop.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/aristocratos/btop/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~m68k ~mips ~ppc ~ppc64 ~riscv ~x86"
+fi
+
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="+gpu"
```

Comment:
any reason to introduce ISUE=gpu? although it's not a regression.. but having some explanation in commit message would be great

Reply:
In any case, btop (and other *top) are frequently used on server systems, so I dislike the idea of enabling USE=gpu by default. In my experience, it is less of a burden to configure USE flags for a laptop than for dozens of headless servers.

Reply:
I totally agree, my thought was that it is enabled by default at the moment, so this would break (?) or switch the default.

Reply:
Actually, I don't see any benefit from adding IUSE=gpu. There is no significant change in compilation time and the dependencies for GPU support are optional, anyways. Also, building with IUSE=gpu doesn't change the libraries btop is linked against:

With USE=gpu
```
❯ sudo objdump -p /var/tmp/portage/sys-process/btop-1.4.0/image/usr/bin/btop | grep NEED
  NEEDED               libstdc++.so.6
  NEEDED               libm.so.6
  NEEDED               libgcc_s.so.1
  NEEDED               libc.so.6
  VERNEED              0x0000000000004b58
  VERNEEDNUM           0x0000000000000004
```

Without USE=gpu
```
❯ sudo objdump -p /var/tmp/portage/sys-process/btop-1.4.0/image/usr/bin/btop | grep NEED
  NEEDED               libstdc++.so.6
  NEEDED               libm.so.6
  NEEDED               libgcc_s.so.1
  NEEDED               libc.so.6
  VERNEED              0x00000000000046c8
  VERNEEDNUM           0x0000000000000004
```

This might fall into the category “controlling runtime dependencies when the package does not link to it”, which is one of the cases where USE flags should _not_ be used, according to the [devmanual](https://devmanual.gentoo.org/general-concepts/use-flags/index.html).